airflow/models/dagbag.py

```diff
@@ -89,7 +89,6 @@ class DagBag(LoggingMixin):
     """
 
     DAGBAG_IMPORT_TIMEOUT = conf.getfloat('core', 'DAGBAG_IMPORT_TIMEOUT')
-    SCHEDULER_ZOMBIE_TASK_THRESHOLD = conf.getint('scheduler', 'scheduler_zombie_task_threshold')
```

Comment:
Hmm, I wonder if we can remove this. It’s not used anywhere (even before this PR) and is probably kept for backward compatibility?

Reply:
Right, I didn't find any reference to that in the codebase, so I just removed as unused code.

But if you believe it should be left there, then of course I can revert this change

Reply:
Let’s revert to be safe (and add a comment saying this is not actually used anywhere and can be removed in Airflow 3).

Reply:
I would be for removing it. There is one other place (local_task_job) where the threshold is used (but directly) so I think this is a leftover from separating this on out. I do not imagine any use of the constant :)